internal/controllers/import_controller.go

```diff
@@ -39,6 +39,7 @@ import (
 
 const (
 	importAnnotation             = "rancher-auto-import"
+	importLabelName              = "cluster-api.cattle.io/rancher-auto-import"
```

Comment:
Why we don't use annotations ?

Reply:
annotations are usually not meant for user interactions, they can even have non human readable format. Labels are designed for users, in addition to that we can do filtering with labelselector if needed, which we can't with annotations.

Reply:
ok, after discussion, it seems that it was already discussed with the team